packages/core/src/js/feedback/FeedbackWidget.styles.ts

```diff
@@ -3,28 +3,28 @@ import type { ViewStyle } from 'react-native';
 import type { FeedbackWidgetStyles } from './FeedbackWidget.types';
 
 const PURPLE = 'rgba(88, 74, 192, 1)';
-const FORGROUND_COLOR = '#2b2233';
-const BACKROUND_COLOR = '#ffffff';
+const FOREGROUND_COLOR = '#2b2233';
+const BACKGROUND_COLOR = '#ffffff';
```

Comment:
For some reason I was blind to this typos 😓 
Thank you for fixing this Krystof 🙇